src/psyclone/transformations.py

```diff
@@ -1474,6 +1513,7 @@ def _validate(self, node_list):
 
         # Now call the general validation checks
         super(OMPParallelTrans, self)._validate(node_list)
+        CheckChildrenList._validate(self, node_list)
 
 
 class ACCParallelTrans(ParallelRegionTrans):
```

Comment:
Although this Issue was originally to do with OpenMP, I expect that the OpenACC implementation that has made it on to master in the interim suffers from the same flaw (since I wrote both of them :-(). Please could you extend your fix to the ACC classes.

I'm also thinking that it would probably be nice if we had some sort of `RegionTrans` base class that could be used by OpenACC/OpenMP/Profiling/PSyKE. Maybe @TeranIvy could consider this as she works on PSyKE?

Reply:
`RegionTrans` base class sounds like a good technical solution, @arporter. Happy to implement it in PSyKE branch!

Reply:
I had a look at the inheritance diagram, and the existing ParallelRegionTrans is at the right location. I'll move the additional tests into _validate of that class.